### PR TITLE
bug fix

### PR DESCRIPTION
**Pull request by douglp**

### CODE REVIEW

File: clustering.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+#print (X_train)
+
+#Vectorisation : -
+tfidfconvert = TfidfVectorizer(analyzer=text_process).fit(X_train)
+
+X_transformed=tfidfconvert.transform(X_train)
+
+# Clustering the training sentences with K-means technique
+
+K = range(1,100)
```
Comment (danny-schwartz7) [inline clustering.py:46]: we don't need a smaller k than the number of datasets we have, so I'd say a hard min is like 6 or 7

File: clustering.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+# read data
+data = util.read_squad("datasets_augmented/indomain_train/squad_subset", args['save_dir'])
+X_train = data['context']
+#print (X_train)
+
+#Vectorisation : -
+tfidfconvert = TfidfVectorizer(analyzer=text_process).fit(X_train)
+
+X_transformed=tfidfconvert.transform(X_train)
```
Comment (danny-schwartz7) [inline clustering.py:42]: What's the dimensionality of this thing?

File: clustering.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+# Clustering the training sentences with K-means technique
+
+K = range(1,100)
+Sum_of_squared_distances = []
+for k in K:
+    km = KMeans(n_clusters=k)
+    km = km.fit(X_transformed)
```
Comment (danny-schwartz7) [inline clustering.py:50]: How long does this take to run?

File: clustering.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+data = util.read_squad("datasets_augmented/indomain_train/squad_subset", args['save_dir'])
+X_train = data['context']
+#print (X_train)
+
+#Vectorisation : -
+tfidfconvert = TfidfVectorizer(analyzer=text_process).fit(X_train)
+
+X_transformed=tfidfconvert.transform(X_train)
+
+# Clustering the training sentences with K-means technique
```
Comment (danny-schwartz7) [inline clustering.py:44]: We might as well do the normalization before checking this in, it won't take much effort

File: clustering.py (hunk 2 above)
Comment (douglp) [reply clustering.py:42]: (3504, 15715)

File: clustering.py (hunk 3 above)
Comment (douglp) [reply clustering.py:50]: It wasn't long actually. I think it finished within an hour on my mac.

File: clustering.py (hunk 4 above)
Comment (douglp) [reply clustering.py:44]: Will do. I'll compare the results with or without normalization, just curious.

**Pull request closed without merging**